operator/Makefile

```diff
@@ -392,19 +345,64 @@ redhat-image-scan:
 
 
 
-bundle_certified_push:
-	source ~/.config/seldon/seldon-core/redhat-image-passwords.sh && \
-		echo $${rh_password_operator_bundle} | docker login -u unused scan.connect.redhat.com --password-stdin
-	docker tag quay.io/seldon/seldon-operator-certified:v${VERSION} scan.connect.redhat.com/ospid-78efc0cc-a4a5-4244-a31d-6820eae1ccb1/seldon-operator-certified:${VERSION}
-	docker push scan.connect.redhat.com/ospid-78efc0cc-a4a5-4244-a31d-6820eae1ccb1/seldon-operator-certified:${VERSION}
-
-update_old_certified_packages:
-	cd packagemanifests-certified && make build_push
+CERTIFIED_OPERATORS_FOLDER=/home/clive/work/seldon-core/redhat/certified-operators
+
+#need to rename package in annotations.yaml to seldon-operator-certified
+update_certified: 
+	cp -r bundle-certified ${CERTIFIED_OPERATORS_FOLDER}/operators/seldon-operator-certified/${VERSION}
+
+
+GIT_REPO_URL=git@github.com:cliveseldon/certified-operators.git
```

Comment:
Seems linked to cliveseldon account

Reply:
Sure this would need to be changed if someone else did it. We could remove and update docs so that it always required to be provided.